amazon_advertising_api/advertising_api.py

```diff
@@ -230,10 +237,15 @@ def create_campaigns(self, data):
                 input.
             :401: Unauthorized
         """
+
+        if campaign_type == 'sb':
```

Comment:
Do you think we could come up with a way to refactor multiple `if campaign_type == 'sb':` statements?

Reply:
fixed